core/kernel.rbs

```diff
@@ -970,7 +969,9 @@ module Kernel : BasicObject
   # For details on `format_string`, see [Format
   # Specifications](rdoc-ref:format_specifications.rdoc).
   #
-  def self?.format: (String format, *untyped args) -> String
+  def self?.sprintf: (string format, hash[Symbol, untyped] keywords) -> String
```

Comment:
Oddly enough, `sprintf` only allows positional or keyword arguments, but not both.